@ndlib/gatsby-theme-marble/src/i18n/withI18nTranslation.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
This is a higher order component that wraps a component in a context. This is used in one place, to wrap the Layout component.